framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -942,6 +942,21 @@ public void reportWarning(
         report(source, Kind.MANDATORY_WARNING, messageKey, args);
     }
 
+    /**
+     * Given an array and two values, creates an array with all the values.
+     *
+     * @param array the array to extend
+     * @param elt1 the first element to add
+     * @param elt2 the second element to add
+     * @return a copy of {@code array}, with two elements added
+     */
+    public static Object[] arrayPlusTwoElements(Object[] array, Object elt1, Object elt2) {
```

Comment:
This is an unexpected place to put a utility method.  I think `org.checkerframework.javacutil.SystemUtil` would be a better location.